front/app/components/FilterSelector/index.tsx

```diff
@@ -131,13 +131,11 @@ export default class FilterSelector extends PureComponent<Props, State> {
       if (isString(title)) {
         newTitle = `${title} (${selection.length})`;
       } else {
-        newTitle = [
```

Comment:
Not sure what was happening here with this randomly generated key, but it was causing errors. This simpler and more readable version didn't